mla/impl/absorbed_cache_compressed_move_elision.py

```diff
@@ -174,7 +174,7 @@ def forward(self, hidden_states_q: torch.Tensor, q_position_ids: torch.LongTenso
             attn_weights, dim=-1, dtype=torch.float32
         ).to(q_nope.dtype)
         attn_output = torch.einsum('bhql,blc->bhqc', attn_weights, compressed_kv)
-        attn_output = torch.matmul(attn_output, out_absorb.mT) # torch.einsum('bhqc,hdc->bhqd', attn_output, out_absorb)
+        attn_output = attn_output = torch.matmul(attn_output.permute(2, 1, 0, 3), out_absorb.mT).permute(2, 1, 0, 3) # torch.einsum('bhqc,hdc->bhqd', attn_output, out_absorb)
```

Comment:
can be changed to `attn_output = torch.einsum('bhqc,hdc->bhqd', attn_output, out_absorb)`